ci/install.sh

```diff
@@ -42,6 +45,17 @@ main() {
            --git japaric/cross \
            --tag $tag \
            --target $target
+
+    if [ $TRAVIS_OS_NAME = linux ]; then
+        echo "AM INSTALLING"
+        sudo apt update
+        sudo apt install -y libsystemd-dev
```

Comment:
Interesting how on linux we target musl and then link to libsystemd, which is installed as a shared library? Have you tested that it works in CI?

Reply:
Good point, I'll write a few smoke tests which will try to write anything to each of supported outputs

Reply:
Check carefully. As far as I understand, the dynamic libraries packaged for Debian are built with the glibc toolchain, so they require ldd and depend on a dynamically loaded libc.so. There may even be a subtle dependency on runtime state managed in the single dynamically linked runtime library. Adding a statically linked musl libc alongside that would break it, even if it is possible.

The [musl FAQ](https://www.musl-libc.org/faq.html) has this to say:

> At present, some glibc-linked shared libraries can be loaded with musl, but all but the simplest glibc-linked applications will fail if musl is dropped-in in place of `/lib/ld-linux.so.2`.

Reply:
Also static libraries built for glibc may require stuff from libc that is not standard and is not present in musl.